documentation/src/main/asciidoc/topics/con_cache_store_sql.adoc

```diff
@@ -1,17 +1,16 @@
 [id='sql-cache-store_{context}']
 = SQL cache stores
 
-SQL cache stores pre-load {brandname} caches from databases, with a supported dialect, through JDBC connections.
-
-You can create two types of SQL cache store:
+SQL cache stores let you load {brandname} caches from database tables.
+{brandname} offers two types of SQL cache store:
 
 Table:: {brandname} loads entries from a single database table.
-Query:: {brandname} uses SQL queries to load entries as well as perform insert, update, and delete operations.
+Query:: {brandname} uses SQL queries to load entries from multiple database tables as well as perform insert, update, and delete operations.
```

Comment:
It can also be used to select individual columns from db rows

Reply:
Added, thanks!